src/components/App/App.js

```diff
@@ -10,9 +10,14 @@ class App extends Component {
     this.props.getPhotos();
   }
   render() {
+    const handleKeyword = (e) => this.props.handleInput(e.target.value);
+    const handleSearch = () => this.props.searchPhotos(this.props.search);
     return (
       <main>
         <h3>Photo Map App</h3>
+        <input onChange={handleKeyword} type="text"/>
+        <button onClick={handleSearch} type="button">Search</button>
```

Comment:
Add type "submit" and make other changes so entering while focussed on input field makes request (clicking not required).